.changeset/itchy-kiwis-unite.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-explore': patch
+---
+
+Extracted generic EntityExplorerContent component so that it is easy to show any component kinds in their own tab in the Explore page.
```

Comment:
```suggestion
Extracted generic `EntityExplorerContent` component so that it is easy to show any component kinds in their own tab in the explore page.
```